Makefile

```diff
@@ -1,26 +1,8 @@
 # Development and testing
 
-all: black isort flake8 mypy test
-
-black:
-	black openslides_backend/ tests/
-
-isort:
-	isort --recursive openslides_backend/ tests/
-
-flake8:
-	flake8 openslides_backend/ tests/
-
-mypy:
-	mypy openslides_backend/ tests/
-
-test:
-	pytest
-
-run-tests:
-	echo "This has to be fixed."
-	OPENSLIDES_BACKEND_RUN_ALL_TESTS=1 pytest
-	# TODO: Run tests inside a container.
+all:
+	$(MAKE) run-cleanup
```

Comment:
Please revert these changes and get my make targets back. I use them every time.

Reply:
Mmh, I think we should be strict about only working inside the docker container. Would it be fine for you if the commands get executed inside the container?

Reply:
No.